### PR TITLE
Updated core/separator block

### PR DESCRIPTION
**Pull request by zainabD**
In this pull request, I updated the native block(core/separator), removed the unwanted (**wide** and **dots**) styles, and added the uds gold style.
Per our last discussion (Walt and I), we decided that we don't need the other petern's dividers that show in storybook [morse-code-white, morse-code-black, topo-white, topo-black, network-white and network-black], as we can use the background section block instead.

image

### CODE REVIEW

**Comment by zainabD** [comment]
This should fix the issue #147

File: sass/theme/_theme.scss [hunk 1]
```diff
@@ -371,3 +371,18 @@ text-align: right;
...
+
+
+/*--------------------------------------------------------------
+Dividers
+--------------------------------------------------------------*/
+.is-style-copy-divider{
+	@extend .copy-divider;
+}
+.block-editor-block-list__layout .wp-block.is-style-copy-divider{
+	margin: 0;
```
Comment (wmcconnell) [inline sass/theme/_theme.scss:383]: I think this line might be the issue with how the gold divider looks in the editor. Setting all margins to 0 removes the built-in 48px top/bottom that the default divider has. Could we get away with only setting `margin-right` and `margin-left` here?

**Review by wmcconnell — CHANGES_REQUESTED** [review]
Everything seems to work, but the gold divider has no top/bottom margin in the Gutenberg editor (looks fine on the page), and sits right on top of whatever is below it (see screen shot). I think it's the result of removing the margins to prevent the existing `auto` margins from always centering things.


-- Screen shot --
Screen Shot 2021-04-23 at 4 16 01 PM

**Review by wmcconnell — APPROVED** [review]
Approving this with the caveat that we may need to go back and look at how we're enqueuing our block styles, as I think the style preview issue may be related to that.

**Pull request merged**